scheduler/pkg/server/pipeline_status.go

```diff
@@ -24,6 +27,71 @@ const (
 	addPipelineStreamEventSource = "pipeline.store.addpipelinestream"
 )
 
+func (s *SchedulerServer) PipelineStatusEvent(ctx context.Context, message *chainer.PipelineUpdateStatusMessage) (*chainer.PipelineUpdateStatusResponse, error) {
+	s.pipelineEventStream.mu.Lock()
+	defer s.pipelineEventStream.mu.Unlock()
+
+	logger := s.logger.WithField("func", "SubscribePipelineStatus")
```

Comment:
Was this logger field name intentional? (it differs from the actual function name where it resides)